elpy.el

```diff
@@ -3633,6 +3633,11 @@ or unless NAME is no callable instance."
      (require 'eldoc)
      (setq eldoc-minor-mode-string nil))
     (`buffer-init
+     ;; avoid interferences between eldoc and company meta frontend
+     (when (member 'elpy-module-eldoc elpy-modules)
+       (set (make-local-variable 'company-frontends)
+            (delq 'company-echo-metadata-frontend company-frontends))
+       (eldoc-add-command-completions "company-"))
```

Comment:
I'm also removing python-indent-dedent-line-backspace here since I found it to play bad with company.

BTW, I submitted an issue to debbugs asking for eldoc to change its whitelist approach for one that just checks if the echo area has no message before writing. This heuristic doesn't depend on a list of commands. Although I'm probably wrong somehow.